server/middlewares/redirectToBaseUrl.js

```diff
@@ -1,5 +1,9 @@
 module.exports = (req, res, next) => {
-  if (`${req.protocol}://${req.get('host')}` !== process.env.ADALITE_SERVER_URL) {
+  if (process.env.HEROKU_IS_REVIEW_APP === 'true') {
```

Comment:
I'm unable to log into the wallet (tried logging with mnemonic), as it fails on CORS, that can be solved by `ADALITE_ENABLE_SERVER_MOCKING_MODE` being set to true (a flag we use to mock requests to tx submission, with the side effect that all requests are being redirected through the heroku server so no CORS issues)

Another problem is that ADALITE_SERVER_URL is used across the app to know what url to call - that can be set with a post-deploy script I believe: https://stackoverflow.com/questions/38087125/set-host-name-as-an-environment-variable-in-heroku-review-app

Reply:
Thanks, will investigate that.